src/k8s-engine/minikube.js

```diff
@@ -265,3 +266,28 @@ async function startAgain(obj) {
 function sleep(delay) {
   return new Promise((resolve) => setTimeout(resolve, delay));
 }
+
+function qq(s) {
+  return /\s/.test(s) ? `"${s}"` : s;
+}
+
+function customizeMinikubeMessage(errorMessage) {
+  let p =/X Exiting due to K8S_DOWNGRADE_UNSUPPORTED:\s*(Unable to safely downgrade existing Kubernetes) (v[\w.]+) (cluster to) (v[\w.]+).*?Suggestion:\s+1\)\s*(Recreate the cluster with.*? by running:)\s+(minikube delete -p rancher-desktop)\s+(minikube start -p rancher-desktop --kubernetes-version=.*?)\n/s;
```

Comment:
Hmm, is there any way we can avoid having a giant line? I see that JS doesn't support `/x`, so that's out…

Do we need to export `LANG=C` when running minikube, to ensure that the messages are in the correct language?

Reply:
My inclination is to leave locale-handling as a YAGNI issue for now, and deal with it if/when it comes up.

I'm more curious about how this behaves on linux and windows